yaml/nats-dep.yml

```diff
@@ -25,3 +25,5 @@ spec:
           - memory
           - --cluster_id
           - faas-cluster
+          - -m
```

Comment:
We didn't expose monitoring before, and I don't think we do for Swarm either, so do we want to do it here by default in helm?

Reply:
Without this, it is basically impossible to debug what is happening inside nats-streaming and any documentation or debug guide will require people to redeploy their OpenFaaS to turn this on and get the information

Reply:
We didn't have it on before?

Reply:
I am disabling it but leaving the configuration commented out, that way it will be easy for someone to re-enable if they need it